igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/service/JenkinsService.java

```diff
@@ -92,10 +95,10 @@ public JenkinsService(
             CircuitBreakerConfig.custom()
                 .ignoreException(
                     (e) -> {
-                      if (e instanceof RetrofitError) {
-                        RetrofitError re = (RetrofitError) e;
-                        return re.getKind() == RetrofitError.Kind.HTTP
-                            && re.getResponse().getStatus() == 404;
+                      if (e instanceof SpinnakerServerException) {
```

Comment:
```suggestion
                      if (e instanceof SpinnakerHttpException) {
```
and take out the instanceof + cast below, right?